draft-johansson-ccwg-rfc8298bis-screamv2.md

```diff
@@ -1279,9 +1291,9 @@ fb_int = 1.0/rate_fb
 
 Feedback should also forcibly be transmitted in any of these cases:
 
-* More than N packets received since last RTCP feedback has been transmitted
+* More than N data units received since last RTCP feedback has been transmitted
```

Comment:
RTCP feedback should likely be just feedback packet.

Reply:
Yes